PocketKit/Sources/Textile/NSAttributedString+Extensions.swift

```diff
@@ -47,13 +48,41 @@ public extension NSAttributedString {
 }
 
 public extension NSMutableAttributedString {
+    enum Constants {
+        static var imageIconSize = CGSize(width: 16, height: 16)
```

Comment:
any reason this cannot be a `let`? Also, since it's just one, maybe we can skip the `enum` and just add a `static` property